tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -221,88 +223,131 @@
                     <ContentControl Template="{ThemeResource LinksTemplate}" />
                 </StackPanel>
 
-
                 <!-- Displays the repos cloned and apps downloaded -->
-                <StackPanel Grid.Column="1" Orientation="Vertical">
+                <StackPanel Grid.Column="1" Orientation="Vertical" x:Name="TaskGroupSections">
+                    <StackPanel.Resources>
+                        <StaticResource x:Key="ExpanderHeaderBackground" ResourceKey="ControlFillColorTransparentBrush" />
+                        <StaticResource x:Key="ExpanderContentBackground" ResourceKey="ControlFillColorTransparentBrush" />

```

Comment:
Do these need to be ThemeResources or does it handle theme change ok as is?

Reply:
Not needed here, but I replaced it with `SolidBrushColor` instead since that change is theme agnostic